internal/frr/config.go

```diff
@@ -69,6 +69,7 @@ type NeighborConfig struct {
 	Port          *uint16
 	HoldTime      *uint64
 	KeepaliveTime *uint64
+	ConnectTime   *uint64
```

Comment:
decided to poinerize the field to do the same as in HoldTime and KeepaliveTime so they will be aligned,
not line decided in metallb (https://github.com/metallb/metallb/pull/2144#discussion_r1457715003)

Reply:
:+1 Makes total sense for it to be consistent here